File: AppGenerateFiles/helpdesk/Model/CaseTable_Mails.cs
using CAPA_DATOS;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;
namespace DataBaseModel {
   public class CaseTable_Mails : EntityClass {
       [PrimaryKey(Identity = true)]
       public int? Id_Mail { get; set; }
       public int? Id_Case { get; set; }
       public string? Subject { get; set; }
       public string? MessageID { get; set; }
       public string? Sender { get; set; }
       public string? FromAdress { get; set; }
       public string? ReplyTo { get; set; }
       public string? Bcc { get; set; }
       public string? Cc { get; set; }
       public string? ToAdress { get; set; }
       public DateTime? Date { get; set; }
       public string? Uid { get; set; }
       public string? Flags { get; set; }
       public string? Estado { get; set; }
       public string? Body { get; set; }
       public string? Attach_Files { get; set; }
       [ManyToOne(TableName = "CaseTable_Case", KeyColumn = "Id_Case", ForeignKeyColumn = "Id_Case")]
       public CaseTable_Case? CaseTable_Case { get; set; }
   }
}

File: AppGenerateFiles/helpdesk/FrontModel/CaseTable_Mails.js
//@ts-check
import { EntityClass } from "../../WDevCore/WModules/EntityClass.js";
import { WAjaxTools } from "../../WDevCore/WModules/WComponentsTools.js";
//@ts-ignore
import { ModelProperty } from "../../WDevCore/WModules/CommonModel.js";
import { CaseTable_Case }  from './CaseTable_Case.js'
class CaseTable_Mails extends EntityClass {
   constructor(props) {
       super(props, 'EntityHelpdesk');
       for (const prop in props) {
           this[prop] = props[prop];
       }
   }
   /**@type {Number}*/ Id_Mail;
   /**@type {String}*/ Subject;
   /**@type {String}*/ MessageID;
   /**@type {String}*/ Sender;
   /**@type {String}*/ FromAdress;
   /**@type {String}*/ ReplyTo;
   /**@type {String}*/ Bcc;
   /**@type {String}*/ Cc;
   /**@type {String}*/ ToAdress;
   /**@type {Date}*/ Date;
   /**@type {String}*/ Uid;
   /**@type {String}*/ Flags;
   /**@type {String}*/ Estado;
   /**@type {String}*/ Body;
   /**@type {String}*/ Attach_Files;
   /**@type {CaseTable_Case} ManyToOne*/ CaseTable_Case;
}
export { CaseTable_Mails }
